acs5countypoverty_child.py

```diff
@@ -0,0 +1,54 @@
+from census_response import getCensusResponse
+import json
+
+def main():
+    #Estimate!!Total!!Population for whom poverty status is determined
+    pop_table = 'S1701_C01_001E'
+    #Estimate!!Below poverty level!!Population for whom poverty status is determined!!AGE!!Under 18 years
+    child_poverty_table = 'S1701_C02_002E'
+    
+    #If additional subdivision are needed
+    #'S1701_C02_003E' = AGE!!Under 18 years!! Under 5 years!!
+    #'S1701_C02_004E' = AGE!!Under 18 years!! 5 to 17 years!!
+
+    subject_table = 'https://api.census.gov/data/2018/acs/acs5/subject?'
+    get_ls = [pop_table, child_poverty_table]
```

Comment:
I think we want to grab the overall poverty with this script too, since its all from the same group.
This is what I found for overall poverty:
`#Estimate!!Below poverty level!!Population for whom poverty status is determined`
`poverty_table = 'S1701_C02_001E'`